bible_routes/v3/version_routes.py

```diff
@@ -82,6 +84,8 @@ async def add_version(
     user_group_ids = [group_id[0] for group_id in user_group_ids_query.all()]
 
     for group_id in user_group_ids:
+        if add_to_groups and group_id not in add_to_groups:
```

Comment:
need to check if add to groups is None

Reply:
I think `if add_to_groups` here should be the check of whether it's `None`. So if it's `None`, you don't skip any group_ids. But if it's a list, you skip those that are not in the list.